packages/data-uri/types/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import { Plugin } from 'rollup';
+
+export default function dataUri(): Plugin;
```

Comment:
nit: It would be nice to have comments for IDEs/VS code
```suggestion
/**
 * A Rollup plugin which imports modules from Data URIs.
 */
export default function dataUri(): Plugin;
```